src/samples/java/com/box/sdk/samples/appusers/webapp/bottom.jsp

```diff
@@ -0,0 +1,11 @@
+        </div>
+      <div class='footer'>
+        <div class='navbar navbar-fixed-bottom custom-footer'>
+          <div class='col-md-12'>
+            <div class='logout'>
+            </div>
+          </div>
+        </div>
+      </div>
+    </body>
+</html>
```

Comment:
All tags should open/close in the same file. I would suggest only keeping the "footer" and nested div tags in this file.